src/main/python/calibration/parse_google_result.py

```diff
@@ -0,0 +1,215 @@
+        return None
+def fourth_attempt(travel_time_str):
+     # for a case like `typically 1 h`
+    regex = r"(?:typically\s){0,1}([\d]{1,2})\sh$"
+    matches = re.finditer(regex, travel_time_str)
+    enum = list(enumerate(matches, start=1))
+    if len(enum) == 1:
+        n, match = enum[0]
+        min_time = int(match.group(1)) * 60 * 60
+        max_time = int(match.group(1)) * 60 * 60
+        return (min_time, max_time)
+    else:
+        return None
+def parse_travel_time(travel_time_str):
+    if isinstance(travel_time_str,float) and math.isnan(travel_time_str):
+        return (float('nan'), float('nan'))
+    temp = first_attempt(travel_time_str)
+    if temp is not None:
+        return temp
+    temp = second_attempt(travel_time_str)
+    if temp is not None:
+        return temp
+    temp = third_attempt(travel_time_str)
+    if temp is not None:
+        return temp
+    temp = fourth_attempt(travel_time_str)
+    if temp is not None:
+        return temp
+    else:
+        raise Exception("Cannot parse '%s' as travel time" % ((travel_time_str)))
+def parse_travel_distance(travel_distance):
+    if isinstance(travel_distance,float) and math.isnan(travel_distance):
+        return float('nan')
+    elif "km" in travel_distance:
+        return float(travel_distance.replace('km', '')) * 1000
+    elif "miles" in travel_distance:
+        # Miles to meteres
+        return float(travel_distance.replace('miles', '')) * 1.60934 * 1000
+    elif "mile" in travel_distance:
+        # Miles to meteres
+        return float(travel_distance.replace('mile', '')) * 1.60934 * 1000
+    elif "ft" in travel_distance:
+        # Miles to meteres
+        return float(travel_distance.replace('ft', '')) * 0.3048
+    elif "m" in travel_distance:
+        return float(travel_distance.replace('m', ''))
+    else:
+        raise Exception("Cannot parse '%s' as travel distance" % ((travel_distance)))
+def normalize(df):
+    df['route_0_travel_distance_meters'] = df['route_0_travel_distance'].apply(lambda x: parse_travel_distance(x))
+    df['route_1_travel_distance_meters'] = df['route_1_travel_distance'].apply(lambda x: parse_travel_distance(x))
+    df['route_2_travel_distance_meters'] = df['route_2_travel_distance'].apply(lambda x: parse_travel_distance(x))
+    df['route_0_travel_time_min'] = df['route_0_travel_time'].apply(lambda x: parse_travel_time(x)[0])
+    df['route_0_travel_time_max'] = df['route_0_travel_time'].apply(lambda x: parse_travel_time(x)[1])
+    df['route_1_travel_time_min'] = df['route_1_travel_time'].apply(lambda x: parse_travel_time(x)[0])
+    df['route_1_travel_time_max'] = df['route_1_travel_time'].apply(lambda x: parse_travel_time(x)[1])
+    df['route_2_travel_time_min'] = df['route_2_travel_time'].apply(lambda x: parse_travel_time(x)[0])
+    df['route_2_travel_time_max'] = df['route_2_travel_time'].apply(lambda x: parse_travel_time(x)[1])
+    return df
+
+def get_input_file(hour):
+  minSeconds = hour * 3600
```

Comment:
`minSeconds` and `maxSeconds ` are mpt following python name patterns. Moreover it is repeated. Extract both to a common constant